os/arch/arm/src/tiva/Make.defs

```diff
@@ -86,12 +86,16 @@ ifneq ($(CONFIG_DISABLE_SIGNALS),y)
 CMN_CSRCS += up_signal_dispatch.c
 CMN_UASRCS += up_signal_handler.S
 endif
-else
+endif
+
 CMN_CSRCS += up_allocateheap.c
+
+ifeq ($(CONFIG_ARM_MPU),y)
+CMN_CSRCS += up_mpuinit.c
 endif
 
 CHIP_ASRCS  =
-CHIP_CSRCS  = tiva_allocateheap.c tiva_start.c tiva_irq.c tiva_gpio.c
+CHIP_CSRCS  = tiva_start.c tiva_irq.c tiva_gpio.c
```

Comment:
What about esp32/esp32_allocateheap.c ??
Do w enot need to remove that file ??

Reply:
It is outside of arch/arm folder. We will not change it for now because our up_allocateheap.c is in in arch/arm/src/common folder.